docs/testing/FE-TEST-CHECKLIST.md

```
@@ -0,0 +1,46 @@
+# Frontend Test Checklist
+
+**Must be 100% Checked Before Merge**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)